check.py

```diff
@@ -123,6 +123,13 @@ def setup_waterfall():
 
 # utilities
 
+def run_command(cmd, stderr=None):
+  print 'executing: ', ' '.join(cmd)
+  proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=stderr)
```

Comment:
Actually never mind, you don't want that when stderr is a pipe too. This is fine.